routing/world_graph.cpp

```diff
@@ -0,0 +1,37 @@
+#include "routing/world_graph.hpp"
+
+namespace routing
+{
+using namespace std;
+
+WorldGraph::WorldGraph(unique_ptr<CrossMwmIndexGraph> crossMwmGraph,
+                       unique_ptr<IndexGraphLoader> loader, shared_ptr<EdgeEstimator> estimator)
+  : m_crossMwmGraph(move(crossMwmGraph)), m_loader(move(loader)), m_estimator(estimator)
+{
+  CHECK(m_loader, ());
+  CHECK(m_estimator, ());
+}
+
+void WorldGraph::GetEdgeList(Segment const & segment, bool isOutgoing, vector<SegmentEdge> & edges)
+{
+  IndexGraph & indexGraph = GetIndexGraph(segment.GetMwmId());
+  indexGraph.GetEdgeList(segment, isOutgoing, edges);
+
+  // TODO remove this return to activate m_crossMwmGraph.
+  return;
+
+  if (!m_crossMwmGraph || !m_crossMwmGraph->IsTransition(segment, isOutgoing))
+    return;
+
+  ASSERT(m_twins.empty(), ());
+  m_crossMwmGraph->GetTwins(segment, m_twins);
+  for (Segment const & twin : m_twins)
+    edges.emplace_back(twin, 0.0 /* weight */);
+  m_twins.clear();
```

Comment:
Is it possible to transform GetTwins() to something like:
```c++
m_crossMwmGraph->ForEachTwin([&edges](Segment const & twin) {
  edges.emplace_back(twin, 0.0 /* weight */);
});
```
This'll prevent you from double-copying.